.gitignore

```diff
@@ -1,4 +1,5 @@
 release
 vendor
 .env
-drone-vault
\ No newline at end of file
+drone-vault
+.idea
```

Comment:
This should be part of your local gitignore, otherwise we will get more IDE specific ignores than regular ignores.